server/src/main/java/org/elasticsearch/search/aggregations/bucket/DocCountProvider.java

```diff
@@ -33,17 +33,25 @@
  */
 public class DocCountProvider {
 
-    private NumericDocValues docCountValues;
+    public static final int defaultValue = DocCountFieldMapper.DocCountFieldType.defaultValue;
```

Comment:
should it be `DEFAULT_VALUE` since it is a constant?

Reply:
Originally, I had it `DEFAULT_VALUE`, but then changed it so that is inline with `DocCountFieldMapper.DocCountFieldType.defaultValue` which is also a constant. 

Changing it for both fields